### ISSUE TITLE
 UNSUPPORTED_NODE: Assertion failed: creator && "Plugin not found, are the plugin name, version, and namespace correct?"

### ISSUE THREAD

**Issue by HireezShanPeng** [NVIDIA/TensorRT#2056]
## Description

I tried to convert my onnx model into trt engine, and I got this error:

```
Loading ONNX file from path onnx_model/segmentation_bert_stn.onnx...
Beginning ONNX file parsing
Assertion failed: creator && "Plugin not found, are the plugin name, version, and namespace correct?"
1
In node 442 (importFallbackPluginImporter): UNSUPPORTED_NODE: Assertion failed: creator && "Plugin not found, are the plugin name, version, and namespace correct?"
Completed parsing of ONNX file
Building an engine from file onnx_model/segmentation_bert_stn_20220601_1860_1.onnx; this may take a while...
758
Completed creating Engine[06/14/2022-11:19:54] [TRT] [E] 4: [network.cpp::validate::2633] Error Code 4: Internal Error (Network must have at least one output)
```

Here is my code below:

```
onnx_model_path = "onnx_model/segmentation_bert_stn.onnx"
tensorrt_path = "trt_engine/demo.trt"

import tensorrt as trt
import common
#['input_ids', 'attention_mask', 'resume_sentence_mask', 'token_type_ids']

def ONNX_build_engine(onnx_file_path, write_engine = False):

    G_LOGGER = trt.Logger(trt.Logger.WARNING)
    explicit_batch = 1 << (int)(trt.NetworkDefinitionCreationFlag.EXPLICIT_BATCH) 
    with trt.Builder(G_LOGGER) as builder, builder.create_network(explicit_batch) as network, trt.OnnxParser(network, G_LOGGER) as parser:
        builder.max_batch_size = batch_size
        config = builder.create_builder_config()
        config.max_workspace_size = common.GiB(2) 
        config.set_flag(trt.BuilderFlag.FP16)
        print('Loading ONNX file from path {}...'.format(onnx_file_path))
        with open(onnx_file_path, 'rb') as model:
            print('Beginning ONNX file parsing')
            parser.parse(model.read())
            print(parser.get_error(0).desc())
            print(parser.num_errors)
            for index in range(parser.num_errors):
                print(parser.get_error(index))
        print('Completed parsing of ONNX file')
        print('Building an engine from file {}; this may take a while...'.format(onnx_file_path))
        profile = builder.create_optimization_profile() 
        profile.set_shape("input_ids", (1,47,24), (1,47,24), (1,47,24))
        profile.set_shape("attention_mask", (1,47,24), (1,47,24), (1,47,24))
        profile.set_shape("token_type_ids", (1,47,24), (1,47,24), (1,47,24))
        config.add_optimization_profile(profile)
        last_layer = network.get_layer(network.num_layers - 1)
        print(network.num_layers)
        # Check if last layer recognizes it's output
        # for i in range(network.num_layers):
        #     print(network.get_layer(i).name)
#         if not last_layer.get_output(0):
#         # If not, then mark the output using TensorRT API
#             network.mark_output(last_layer.get_output(0))
        engine = builder.build_engine(network, config)
        print("Completed creating Engine")
        if write_engine:
            engine_file_path = 'correction_fp16.trt'
            with open(engine_file_path, "wb") as f:
                f.write(engine.serialize())
#         return engine
ONNX_build_engine(onnx_model_path)
```

## Environment

**TensorRT Version**: 8.2.2
**NVIDIA GPU**: M60
**CUDA Version**: 11.1
**CUDNN Version**: 8005
**Python Version (if applicable)**: 1.9.0
**Baremetal or Container (if so, version)**: 

Hope your suggestions on this, THANKS

**Comment by nvpohanh** [comment]
@HireezShanPeng Could you share your ONNX file? I think it contains some ops that TRT does not support yet.

**Comment by nvpohanh** [comment]
Closing due to >14 days without activity. Please feel free to reopen if the issue still exists. Thanks

**Comment by frankvp11** [comment]
Hi i'm having a similar issues- is it possible to reopen this? The code i'm using the build_engine.py from TensorRT/samples/python/detectron2, and I can give you the onnx file im using. My TensorRT version is 8.2.1, python3.6.9 and cuda 10.2

**Comment by yehangyang** [comment]
@nvpohanh ,
Does TensorRT have any tools to help check and print unsupported operations?

**Comment by RajUpadhyay** [comment]
Hi everyone, 
I am relatively new to onnx and tensorrt conversion and I am in a quite a fix.
I used detectron2's **_mask_rcnn_R_101_FPN_3x.yaml_** and converted it to an onnx file using _**tools/deploy/export_model.py**_ using an image of dimensions 1334x1334.

But whenever I try to further implement it with tensorrt, it throws an error:
In node 19 (importFallbackPluginImporter): UNSUPPORTED_NODE: Assertion failed: creator && "Plugin not found, are the plugin name, version, and namespace correct?"

and also a warning
[02/21/2023-09:52:44] [TRT] [W] onnx2trt_utils.cpp:369: Your ONNX model has been generated with INT64 weights, while TensorRT does not natively support INT64. Attempting to cast down to INT32.

Can you please tell me what am I doing wrong? Thank You.

![model onnx](https://user-images.githubusercontent.com/38916111/220221479-c2519ae0-a800-4710-aeda-dbbfc7a245f2.svg)

**Comment by frankvp11** [comment]
I'm assuming the model graph you have provided is correct. Something to check into is if your tensorrt is at the latest version. You could also check a different conversion method.

**Comment by frankvp11** [comment]
As for the warning, I wouldn't be concerned about it

**Comment by RajUpadhyay** [comment]
Thank you for the reply!

I believe it is correct since I validated them with a few checker codes provided by onnx.
I converted it directly into onnx rather than caffe since caffe has been deprecated.
But my tensorrt version is 8.4.3.1 which is not the latest so I'll do something about it.

Also, seeing the error, do you believe writing custom plugin might do it?
If yes then could you point me to some samples since I have no experience with them.

Thank You.

**Comment by frankvp11** [comment]
I don't believe you need to write custom plugins for the conversion. I wrote an article in September on how to do this (converting) with the mask rcnn 50, which I will link with this.  However looking through the tensorrt repo they don't have an example for the 101 fpn, which makes me think it might not be possible (I'm not a tensorrt dev so idk that) but that's something else to consider. If all else fails, I genuinely wouldn't know what to do and you'd need to wait for official support.
Article: https://medium.com/p/2cd710954ad3

**Comment by RajUpadhyay** [comment]
Ok, I will check it out!
Thanks
